### DISCUSSION TITLE
Cert import automatically

### DISCUSSION THREAD

**Discussion by stanthewizzard** [ZoeyVid/NPMplus#933] [Q&A] [answered]
Hello (me again lol)

Is there a way to import automatically a cert that is generated elsewhere ?
Manual import does work

Thanks

**Comment by Zoey2936** [comment]
Hello, currently not. But you can open the dev console while uploading a cert and use this data to make a custom request to upload certs automatically

**Reply by stanthewizzard** [reply]
you are everywhere lol
THANKS
Is there a tuto for the dev console ?

**Reply by Zoey2936** [reply, marked as answer]
You open it (most time F12), you open the network tab, enable "keep log entries" (not sure hiw the correct name is but search for something like this), upload your cert while having the dev tools open, right click the post/put request(s) and search fir copy as curl, then you can modify this command(s) to fit your needs

**Reply by Zoey2936** [reply]
(What do you mean with "you are everywhere"?)

**Reply by stanthewizzard** [reply]
> (What do you mean with "you are everywhere"?)

that you are very quick to answer all of my question.
I don't know how to thank you

understand the curl command
clever
thanks again
